packages/phone-number-privacy/combiner/test/end-to-end/get-blinded-sig.test.ts

```diff
@@ -29,6 +31,16 @@ describe('Running against a deployed service', () => {
     console.log('ODIS_COMBINER_SERVICE_URL: ' + process.env.ODIS_COMBINER_SERVICE_URL)
     console.log('ODIS_BLOCKCHAIN_PROVIDER: ' + process.env.ODIS_BLOCKCHAIN_PROVIDER)
   })
+
+  xit('Service is deployed at correct version', async () => {
```

Comment:
checking my understanding: this is currently excluded since the status endpoint isn't working as expected, and will be re-added back once that endpoint is fixed? Or is this supposed to be manually run?

Reply:
Your understanding is correct, the status endpoint is broken. I'll add a comment to explain this